wonjong/app.js

```diff
@@ -96,6 +96,39 @@ app.get("/user/post", async (req, res) => {
   res.status(200).json({ data: allUsersPostsViews });
 });
 
-app.listen(PORT, function () {
+app.patch('/post/patch', async (req, res) => {
```

Comment:
이미 HTTP method의 patch로 어떤 행위를 할 것인지 정해지기 때문에 endpoint에서 patch는 제거해 주셔도 될 것 같습니다.
```suggestion
app.patch('/post', async (req, res) => {
```